public/app/features/alerting/unified/hooks/useAbilities.ts

```diff
@@ -28,6 +28,7 @@ export enum AlertmanagerAction {
   ViewNotificationTemplate = 'view-notification-template',
   UpdateNotificationTemplate = 'edit-notification-template',
   DeleteNotificationTemplate = 'delete-notification-template',
+  CanReadSecrets = 'can-read-secrets',
```

Comment:
It seems to me this should refer to an action we want to perform in the UI rather than be a direct proxy to the permission
Could we rename it to something like `ExportContactPoints`?
@gillesdemey wdyt?

Reply:
It looks like we already. have `ExportContactPoints` ability. If I understand correctly `CanReadSecrets` should only be used as the `decrypt` parameter when we send the export request to the API. 
From that perspective `CanReadSecrets` doesn't work like a standard permission and maybe having it in this hook is not the best idea

Reply:
i agree

Reply:
Following a conversation with @konrad147 we agreed renaming this Ability to `DecryptSecrets`, as it makes more sense as a name for an action/ability.

Reply:
+1 for renaming it – you've already figured out the reason why it's a separate action but I like `DecryptSecrets`